.rubocop.yml

```diff
@@ -1 +1,19 @@
 inherit_from: .rubocop_todo.yml
+AllCops:
+  Exclude:
+    - 'test/dummy/**/*'
+    - 'tmp/**/*'
+    - 'spec/fixtures/**/*'
+    - 'vendor/**/*'
+  TargetRubyVersion: 2.3
+
+# Apparently a thing in Rubocop for Ruby 2.3+ in prep for Ruby 3
```

Comment:
Yeah I saw something when I was googling the rubocop error, I do believe there's a option you can give it when you run rubocop that'll add the line to every file (`--auto-correct`, I think?). I actually found that exact page when googling the topic a few minutes ago :P